python/cuvs/cuvs/test/test_brute_force.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pytest
+from pylibraft.common import device_ndarray
+from scipy.spatial.distance import cdist
+
+from cuvs.neighbors import brute_force
+
+
+@pytest.mark.parametrize("n_index_rows", [32, 100])
+@pytest.mark.parametrize("n_query_rows", [32, 100])
+@pytest.mark.parametrize("n_cols", [40, 100])
+@pytest.mark.parametrize("k", [1, 5, 32])
```

Comment:
Why these values? Is 5 testing something that 32 isn't? Note that this is a general question for all the pytests including the ones I added from pylibraft, so we could do a follow up work for deciding this.

Reply:
I believe 5 was added for making sure we include a k value thats not a power of 2 - 

Fwiw, this test was copied from the the one in pylibraft - which is what originally included these parameters